LoopKitUI/ViewModels/PreferredGlucoseUnit.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  PreferredGlucoseUnit.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2021-03-08.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+public class PreferredGlucoseUnit: ObservableObject {
```

Comment:
[nit] @rickpasetto has proposed using an "Observable" suffix on this. I added it to the Thursday meeting agenda, but we can discuss now.

Reply:
If we add the suffix, then I think we should rename the property from `unit` to `preferredGlucoseUnit`.